### DISCUSSION TITLE
Silent Install Not behaving as expected.

### DISCUSSION THREAD

**Discussion by heretolearn24** [mozilla/policy-templates#1167] [Q&A]
Currently have x86 installed via .bat using:

%~dp0Fire86.exe /s

This installs as expected and silently.

But when I try to do the same for x64 .exe

%~dp0Fire64.exe /s

It installs but not silently, we get a popup. Is there something I am missing?

**Comment by mkaply** [comment]
I would open this up as a bugzilla bug.

what is the exact pop-up?

**Reply by heretolearn24** [reply]
Hello, this is what we get. 
![2024-11-04 08_59_13-popup - Paint](https://github.com/user-attachments/assets/0edbdd9d-f437-46fc-9b4c-55384f695b45)

**Reply by mkaply** [reply]
That's the stub installer not the full installer. where are you downloading it from?

**Reply by heretolearn24** [reply]
From here: 
https://www.mozilla.org/en-US/firefox/all/desktop-release/

**Comment by mkaply** [comment]
Sorry, dropped the ball on this.

So I just double checked and I get the full installer from this page:

https://www.mozilla.org/en-US/firefox/all/desktop-release/

whether I pick 32 bit or 64 bit. (I can tell because the file sizes are around 60-70 meg.

What specific sequence did you pick?
